src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private Object[] keyArray;
+    private Object[] valueArray;
+    private int arraySize;
```

Comment:
Array size is 10, isn't it?)
```suggestion
    private int size;
```
or
```suggestion
    private int storageSize;
```